src/jit/codegenarm64.cpp

```diff
@@ -1613,7 +1613,7 @@ void CodeGen::genCodeForLclVar(GenTreeLclVar* tree)
         instruction ins  = ins_Load(targetType);
         emitAttr    attr = emitTypeSize(targetType);
 
-        attr = emit->emitInsAdjustLoadStoreAttr(ins, attr);
+        attr = emitter::isVectorRegister(tree->gtRegNum) ? attr : emit->emitInsAdjustLoadStoreAttr(ins, attr);
```

Comment:
You should review the other calls to emitInsAdjustLoadStoreAttr
as I believe that all of them need to check for a floating point register target.

A better test than isVectorRegister() too use in most cases is: `varTypeIsFloating(targetType)`

Reply:
I reviewed all use.  The remaining ones were always using integer registers, but I'll check again to be sure.

The one case where this was already handled used `isVectorRegister`.  I'll see if I can switch them all to `varTypeIsFloating`